TheWarWithin/HunterBeastMastery.lua

```diff
@@ -1245,9 +1271,11 @@ spec:RegisterHook( "reset_precast", function()
         end
     end
 
-    if now - action.resonating_arrow.lastCast < 6 then applyBuff( "resonating_arrow", 10 - ( now - action.resonating_arrow.lastCast ) ) end
+    if covenant.kyrian then
```

Comment:
Merge the nested `if` -- `if covenant.kyrian and now - action.resonating_arrow.lastCast...`